testing/trino-tests/src/test/java/io/trino/execution/TestEventListenerBasic.java

```diff
@@ -1140,6 +1146,69 @@ public Object[][] setOperator()
+                        ImmutableList.of(new JsonRenderedNode(
+                                "98",
+                                "Limit",
+                                ImmutableMap.of("count", "10", "withTies", "", "inputPreSortedBy", "[]"),
+                                ImmutableList.of(typedSymbol("symbol_1", "double")),
+                                ImmutableList.of(),
+                                ImmutableList.of(),
+                                ImmutableList.of(new JsonRenderedNode(
+                                        "171",
+                                        "LocalExchange",
+                                        ImmutableMap.of(
+                                                "partitioning", "[connectorHandleType = SystemPartitioningHandle, partitioning = SINGLE, function = SINGLE]",
+                                                "isReplicateNullsAndAny", "",
+                                                "hashColumn", "[]",
+                                                "arguments", "[]"),
+                                        ImmutableList.of(typedSymbol("symbol_1", "double")),
+                                        ImmutableList.of(),
+                                        ImmutableList.of(),
+                                        ImmutableList.of(new JsonRenderedNode(
+                                                "138",
+                                                "RemoteSource",
+                                                ImmutableMap.of("sourceFragmentIds", "[1]"),
+                                                ImmutableList.of(typedSymbol("symbol_1", "double")),
+                                                ImmutableList.of(),
+                                                ImmutableList.of(),
+                                                ImmutableList.of()))))))),
+                "1", new JsonRenderedNode(
+                        "137",
+                        "LimitPartial",
+                        ImmutableMap.of(
+                                "count", "10",
+                                "withTies", "",
+                                "inputPreSortedBy", "[]"),
+                        ImmutableList.of(typedSymbol("symbol_1", "double")),
+                        ImmutableList.of(),
+                        ImmutableList.of(),
+                        ImmutableList.of(new JsonRenderedNode(
+                                "0",
+                                "TableScan",
+                                ImmutableMap.of(
+                                        "table", "[table = catalog_1.schema_1.table_1, connector = tpch]"),
+                                ImmutableList.of(typedSymbol("symbol_1", "double")),
+                                ImmutableList.of("symbol_1 := column_2"),
+                                ImmutableList.of(),
+                                ImmutableList.of()))));
+        assertThat(event.getMetadata().getJsonPlan())
```

Comment:
We should anonymize both `jsonPlan` and `plan`. Could you create an issue + add TODO?

Reply:
We are anonymizing both plan and jsonPlan. It's just there's no test for `plan` in `TestEventListenerBasic`. In general, there's no good testing of the text plan.